movingpandas/geometry_utils.py

```diff
@@ -42,7 +56,7 @@ def _measure_distance(point1, point2, spherical=False):
     if spherical:
         return measure_distance_spherical(point1, point2)
     else:
-        return measure_distance_euclidean(point1, point2)
+        return measure_distance_geodesic(point1, point2)
```

Comment:
Geodesic distance should replace spherical distance. Euclidean distance should remain the default if the CRS is not spherical